services/sensorhub-service-video/build.gradle

```diff
@@ -13,6 +13,11 @@ osgi {
   manifest {
     attributes('Bundle-Vendor': 'Sensia Software LLC')
     attributes('Bundle-Activator': 'org.sensorhub.impl.service.sos.video.Activator')
+    attributes('-exportcontents': 'org.sensorhub.impl.service.sos.video')
+    attributes('Import-Package': '''
+      !javax.crypto.*,!javax.imageio.*,!javax.xml.*,!java.*, !org.w3c.*, !org.xml.*,
+      org.osgi.service.repository;resolution:=optional,
```

Comment:
Don't think we need to import "org.osgi.service.repository;resolution:=optional"

Reply:
608c1b825083a95ae8b6017ca7d5ffb482ebd176